### PR TITLE
Fix more issues

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

File: src/styl/responsive/tablet-portrait.styl [hunk 1]
```diff
@@ -47,5 +46,6 @@ $header-height= 50px
 
 #map-holder
   iframe
-    width: 100%
+    position: absolute
```
Comment (varffs) [inline src/styl/responsive/tablet-portrait.styl]: Adding positioning this breaks the layout. On my local with `absolute` the element has no height and doesnt display.

**Review by varffs — CHANGES_REQUESTED** [review]
Not sure about the reasons for map width changes so dont quite understand but currently this isn't rendering correctly.

Could you put the mailchimp embed code somewhere on the trello too please.

File: src/styl/responsive/tablet-portrait.styl (hunk 1 above)
Comment (mikeyrayvon) [reply src/styl/responsive/tablet-portrait.styl]: its so it fills the container on safari. ive tested across browsers. not sure why it isnt displaying for you

**Comment by mikeyrayvon** [comment]
mailchimp / map code in trello @varffs

**Comment by varffs** [comment]
The `#mc_embed_signup div.response` styling is a bit off really. I'm guessing default. I'll just do a minor tweak

**Comment by mikeyrayvon** [comment]
theres a card about that in Trello @varffs leave it as-is. i'm going to do a design for it.

**Comment by varffs** [comment]
The map bug persists. If you remove the mailchimp signup code you will surely lose the map embed but only as it has no height

**Comment by varffs** [comment]
And removing the absolute position from the iframe in Safari debugger makes the map come back

**Comment by mikeyrayvon** [comment]
then give the map a min-height

**Comment by mikeyrayvon** [comment]
`#map-holder {
min-height: 450px;
}`
👍

**Pull request merged**